{"bug_class": "PROVER", "kind": "ERROR", "bug_type": "NULL_DEREFERENCE", "qualifier": "object `type` last assigned on line 71 could be null and is dereferenced by call to `newInstance(...)` at line 78.", "severity": "HIGH", "visibility": "user", "line": 78, "column": -1, "procedure": "java.lang.Object[] ArrayUtils.mergeArrays(java.lang.Object[][])", "procedure_id": "org.panda_lang.utilities.commons.ArrayUtils.mergeArrays(java.lang.Object[][]):java.lang.Object[].e49264932b2f8d655f78940922290595", "procedure_start_line": 69, "file": "panda-utilities/src/main/java/org/panda_lang/utilities/commons/ArrayUtils.java", "bug_trace": [{"level": 0, "filename": "panda-utilities/src/main/java/org/panda_lang/utilities/commons/ArrayUtils.java", "line_number": 69, "column_number": -1, "description": "start of procedure mergeArrays(...)"}, {"level": 0, "filename": "panda-utilities/src/main/java/org/panda_lang/utilities/commons/ArrayUtils.java", "line_number": 70, "column_number": -1, "description": ""}, {"level": 0, "filename": "panda-utilities/src/main/java/org/panda_lang/utilities/commons/ArrayUtils.java", "line_number": 71, "column_number": -1, "description": ""}, {"level": 0, "filename": "panda-utilities/src/main/java/org/panda_lang/utilities/commons/ArrayUtils.java", "line_number": 73, "column_number": -1, "description": "Taking false branch"}, {"level": 0, "filename": "panda-utilities/src/main/java/org/panda_lang/utilities/commons/ArrayUtils.java", "line_number": 78, "column_number": -1, "description": ""}], "key": "ArrayUtils.java|mergeArrays|NULL_DEREFERENCE", "node_key": "adc47e00a8d1d4ea5f6f6c1bea7e7c69", "hash": "192ac83bacdf12f847a3ddee46b9526f", "bug_type_hum": "Null Dereference", "censored_reason": ""}